src/Dialogs/AbstractPropertiesDialog.vala

```diff
@@ -120,21 +118,21 @@ protected abstract class Files.View.AbstractPropertiesDialog : Granite.Dialog {
             };
 
             file_img.add_overlay (file_icon);
-            file_img.add_overlay (emblem_grid);
+            file_img.add_overlay (emblem_box);
 
             layout.attach (file_img, 0, 0, 1, 1);
         } else {
             layout.attach (file_icon, 0, 0, 1, 1);
         }
     }
 
-    protected void add_section (Gtk.Stack stack, string title, string name, Gtk.Container content) {
+    protected void add_section (Gtk.Stack stack, string title, string name, Gtk.Widget content) {
         if (content != null) {
             stack.add_titled (content, name, title);
         }
 
         /* Only show the stack switcher when there's more than a single tab */
-        if (stack.get_children () != null) {
+        if (stack.get_pages ().get_n_items () > 1) {
             stack_switcher.show ();
         }
```

Comment:
Since we removed a `no_show_all` we probably need to rework this. I think currently the StackSwitcher will always show